lib/shell.js

```diff
@@ -1,12 +1,10 @@
 import util from 'node:util';
-import sh from 'shelljs';
+import sh from 'node:child_process';
 import { execa } from 'execa';
 import { format } from './util.js';
 
 const debug = util.debug('release-it:shell');
 
-sh.config.silent = !debug.enabled;
```

Comment:
This functionality is pretty useful for a few reasons:

- Clean output in release-it output for users
- Clean output in release-it output during test runs

Output is disabled by default, and can be enabled using `NODE_DEBUG=release-it:*` as documented here: https://github.com/release-it/release-it?tab=readme-ov-file#troubleshooting--debugging

Any chance this could still be worked in?

Reply:
From what I see, the config here is only for `lib/shell.js` right? and that hindered `shelljs` to pipe its `stdout` to `process.stdout`. But in `lib/shell.js`, you already expect `stdout`. So either I am not following the logic here, or I need to ignore `stdout` when `!debug.enabled`

Reply:
which brings up the question, you use `execa` as well. wouldn't it be wise to replace that as well (in another PR?)

Reply:
Ah, ok. This will apply to exec-stuff _outside_ of `lib/shell.js` from what I can tell, since `execStringCommand` only displays output on log level verbose, which is debug. Right?

Reply:
I think I have the solution. Stay tuned

Reply:
That sounds about right indeed. I think the behavior can be seen most clearly when comparing output of running the tests in `main` with running them in this branch, as in `main` there's almost no output other than what the test runner Ava logs. Same when actually running release-it as a user (I sometimes do this during development using `npm link`).